hw/hw12/mailroom.py

```diff
@@ -0,0 +1,231 @@
+# records of donation
+D_DONERS = {'John Smith': [320],
+            'Mary Simpson': [100, 150, 200],
+            'Chris Finch': [400],
+            'Larry Bookman': [150, 200],
+            'Victoria Black': [60]
+            }
```

Comment:
Some linters want the closing `}` to be flush with the variable name, and that's the style I prefer, but it's just a matter of opinion and there's no right answer.